app/assets/javascripts/table-select-all.es6

```diff
@@ -25,6 +26,12 @@ $(document).ready(function() {
       return false;
     });
 
+    $('.select-clear-all').keyup(function(ev) {
+      if (ev.which==13 || ev.which==32)  { //on space or return, the column is sorted
```

Comment:
It would be clearer and avoid magic numbers if 13 and 32 were assigned to variables:
```javascript
const spaceKey = 13
const returnKey = 32
if (ev.which == spaceKey || ev.which == returnKey) {
```
Also the default behavior is for a space key press to trigger `click` (see [this](https://stackoverflow.com/questions/27878940/spacebar-triggering-click-event-on-checkbox) for example). Will this cause click to be triggered twice? Is that a problem?

Reply:
FYI you can use `return false` to prevent the default behaviour of the event.

Reply:
I added this in after trying the space bar and return key and seeing it didn't work.  I think this is because I have changed the link from an anchor to a normal header - this was done to increase the click area as inplied in the ticket and so styles could be reused.  

It should not affect the tick boxes themselves, they should still exhibit default behaviour.  

I'll push this to UAT and do my own checks cross-browser as well, just to be sure.

Reply:
**Cross-browser testing**: Edge submitted page on sort.  I'll re-do to inculde `return false` as recommended.

Reply:
Nicely done.